src/Profile/Profile.module.scss

```diff
@@ -0,0 +1,4 @@
+h2{
+    color: #1877f2;
+    font-weight: 900;
+}
```

Comment:
This will affect every h2, not just the ones in the profile. Is that what you were trying to do?